test/integration/graql/reasoner/ReasoningIT.java

```diff
@@ -20,8 +20,6 @@
 import com.google.common.collect.Iterables;
```

Comment:
Comments have been added to the tests so the reviewer is able to figure out where they were migrated to.
If unspecified, they were migrated to `relation-inference.feature` in verification.